app/services/create_user.rb

```diff
@@ -9,29 +9,11 @@ def initialize(user:, organisation:)
   def call
     result = Result.new(true)
 
-    User.transaction do
-      user.organisation = organisation
-      user.save
-      begin
-        CreateUserInAuth0.new(user: user).call
-      rescue Auth0::Exception => e
-        result.success = false
-        result.error_message = extract_auth0_error_message(e)
-        Rails.logger.error("Error adding user #{user.email} to Auth0 during CreateUser with: #{result.error_message}")
-        raise ActiveRecord::Rollback
-      end
-    end
+    user.organisation = organisation
+    user.password = SecureRandom.uuid
+    result.success = user.save
 
     SendWelcomeEmail.new(user: user).call if user.persisted?
```

Comment:
Perhaps should read `if result.success?`

Reply:
Opened https://trello.com/c/fODkynU1/2452-remove-usernotauthorised-paraphernalia to deal with this, as it's not as simple as it first appears.